### ISSUE TITLE
Can't export contacts

### ISSUE THREAD

**Issue by SNvMK** [SimpleMobileTools/Simple-Contacts#716]
# Issue

Cannot export contacts to .vcf file, because app is crashing

# Steps to reproduce

1. Click "Export to .vcf-file"
2. Select any source (even all)
3. Click "OK"
4. Observe the issue

# Expected behaviour

Contacts are got exported to .vcf file

# Actual behaviour

App is crashing

# Additional info

Simple Contacts version: 6.15.1 (from F-Droid)
ROM version: LineageOS 18.1 (Android 11)
Device: Xiaomi Redmi 4A

**Comment by tibbi** [comment]
If you select some contacts by long pressing and using Share, does it work? Try sending yourself the .vcf file via email or so.

**Comment by SNvMK** [comment]
yeah, it worked. but i think that export bug should also be solved, or just removed

**Comment by tibbi** [comment]
It might be some specific contact breaking it. Try long pressing one contact, then using Select all and Share again. Does it work?

**Comment by SNvMK** [comment]
@tibbi yep it works

**Comment by tibbi** [comment]
okay, guess Ill take a look at the crashdumps and see what could be wrong there.
